

W Makefile'u aic7xxx wlaczone jest -Werror, co powoduje, ze do skompilowania
tegoz na alphie potrzebuje takiej brzydkiej latki:

***************************************************************
--- linux-2.4.21/drivers/scsi/aic7xxx/aic79xx_osm_pci.c.orig	Fri Jul 11 00:40:03 2003
+++ linux-2.4.21/drivers/scsi/aic7xxx/aic79xx_osm_pci.c	Fri Jul 11 00:43:22 2003
@@ -52,9 +52,12 @@
 					const struct pci_device_id *ent);
 static int	ahd_linux_pci_reserve_io_regions(struct ahd_softc *ahd,
 						 u_long *base, u_long *base2);
+#ifdef MMAPIO
 static int	ahd_linux_pci_reserve_mem_region(struct ahd_softc *ahd,
 						 u_long *bus_addr,
 						 uint8_t **maddr);
+#endif
+
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(2,4,0)
 static void	ahd_linux_pci_dev_remove(struct pci_dev *pdev);
 
@@ -271,6 +274,7 @@
 	return (0);
 }
 
+#ifdef MMAPIO
 static int
 ahd_linux_pci_reserve_mem_region(struct ahd_softc *ahd,
 				 u_long *bus_addr,
@@ -318,6 +322,7 @@
 		error = ENOMEM;
 	return (error);
 }
+#endif
 
 int
 ahd_pci_map_registers(struct ahd_softc *ahd)
***************************************************************

-- 
=======================================================================
  Andrzej M. Krzysztofowicz               ankry@mif.pg.gda.pl
  phone (48)(58) 347 14 61
Faculty of Applied Phys. & Math.,   Gdansk University of Technology
